elections/tac-and-scir-election-process.md

```diff
@@ -0,0 +1,66 @@
+Eligible community members, according to the same criteria defined above for voter eligibility, can self-nominate using the form below.
+
+[TAC Candidate Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLSdxUbbfJGgjU_cZ_uGvUm-5ePoXu7q2pGiNyH2G8euwpn23Sw/viewform)
+
+
+## SCIR Self-Nomination Information and Process
+
+Since early in its existence, the OpenSSF Governing Board has sought to ensure it gets adequate input from voices in the software security community who would otherwise not be at the table.
+We seek candidates for the Security Community Individual Representative (SCIR) who can represent those voices, while also being a subject matter expert in the field with their own set of perspectives.
+Familiarity with the different OpenSSF working groups and projects, and being able to dedicate the time to be sufficiently informed on the issues that arise in our monthly calls and ongoing discussions, is highly desired.
+It is also highly desired, but not required, that the SCIR be an eligible OpenSSF community member by the criteria defined above.
+
+[SCIR Candidate Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLScoabVVjW1kvTNcKXhTEdvxQd13Caql-I36MOixHVO1dLwIiQ/viewform)
+
+
+## Election Process
+
+The Election Process will be monitored by Election Officials, who
+* review nominees and confirm their eligibility,
+* approve the election timeline, and
+* review the ballot request form and validate voter eligibility.
+
+Election Officials are eligible community members who have volunteered to oversee and help ensure the integrity of the election process. By volunteering in this role, they recuse themselves from running for a TAC or SCIR position but they still are allowed to vote in the election.
+
+Candidates for each election will be reviewed by the Election Officials to ensure their eligibility per the guidelines above. At the end of the validation period, the complete list of candidates will be published in this repo.
+
+Similarly, the pool of eligible voters will be reviewed by Election Officials to ensure that all voters meet the eligibility criteria established above If a ballot requester is considered ineligible, an Election Official will follow up directly with the individual.
+
+If there are more nominees than positions for either the TAC or the SCIR, the election will be conducted using an electronic voting tool.
+The [Single Transferable Vote](https://en.wikipedia.org/wiki/Single_transferable_vote) method will be used to vote for TAC members as there are multiple candidates for multiple positions.
+Eligible voters will receive an invitation to access their ballot from Opavote.
+The ballot will include the option to vote for both the TAC and SCIR.
+
+Questions related to the nomination and/or election process should be directed to [operations@openssf.org](mailto:operations@openssf.org).
+
+
+## 2023 Election Timeline
```

Comment:
To avoid letting this get out of date, perhaps we call this section "Standard Election Timeline" and use the 2023 dates by way of example. So next year we can post a timeline that suits how the calendar falls without it being something we have to remember to update here. Just a thought.

Reply:
Yes, this is an option, too. My original thinking was to actually use this file to publish the timelines every year, i.e., in 2024 we would add a new section, but I am not overly opinionated about this. We can just keep this as a reference so that we know how much time we want to allocate to each period.

Reply:
Yeah, I'd like to see this be something that is adopted, and then used over time, so shifting this to a standard timeline is best.

Reply:
I think we all have seen, from experience, this kind of info to go stale. I like the pattern of adopting a standard timeline where we are committing to the number of days in each phase (nominations are open for 14 business days, say) - and we can use the public calendar as well to make sure it's noted when the TAC should be kicking off an election process on an annual basis

Reply:
ok, good for me. I'll update the text accordingly.